doc/manual/source/developer_guide/ModificationIntro.rst

```diff
@@ -149,9 +153,6 @@ Mailing List <http://groups.google.com/group/enzo-users>`_.
 How To Use Branching
 --------------------
 
-[This should be updated to refer to Git, not Mercurial]
-
-
 .. warning:: In most cases, you do *not* need to make a new named branch!  Do
    so with care, as it lives forever.
 
```

Comment:
I've reverted this file to its previous (outdated) state, except for the new note at the top. I don't think I know git well enough to fix all the sections in this doc, so instead of changing some portions to git and leaving some as mercurial -- which would be confusing to other developers -- I left everything as mercurial for someone else to fix. Sorry to put the work on someone else.